mapApp/static/mapApp/js/edit_hazards.js

```diff
@@ -116,7 +116,10 @@ $(function(){
   geofences = geojsonPolygonMarker(geofences);
   geofences.setStyle({color: "blue"});
   geofences.addTo(map);
-  map.fitBounds(geofences);
+  if (geofences.getBounds().isValid()){
```

Comment:
Is geofences.getBounds() guranteed to always return a result? Just making sure we won't potentially call isValid() on an object that is None.